Makefile

```diff
@@ -1,189 +1,608 @@
-MAKEFLAGS+=-r -j
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Does it make sense to check this in or should it be removed and `.gitignore`-ed?

Reply:
.gitignore